fdbclient/include/fdbclient/StorageCheckpoint.h

```diff
@@ -128,6 +128,11 @@ class hash<CheckpointMetaData> {
 
 // A DataMoveMetaData object corresponds to a single data move.
 struct DataMoveMetaData {
+private:
+	// a deprecated field only for downgrade to or upgrade from 71.2 compatible;
+	KeyRange range;
```

Comment:
I think it would be nice to name this `deprecated_range` or something similar to make it super clear everywhere in the code that we shouldn't use this for future changes.